src/test/java/duke/parser/CommandParserTest.java

```diff
@@ -0,0 +1,110 @@
+package duke.parser;
```

Comment:
It appears JavaDoc comments for your test classes have yet to be added. Do remember to add them in when fine-tuning your file! 👍🏻